sdk/src/azure/core/az_mqtt5_connection.c

```diff
@@ -40,6 +40,52 @@ AZ_NODISCARD az_mqtt5_connection_options az_mqtt5_connection_options_default()
   };
 }
 
+static az_result _az_mqtt5_connection_event_relay_process_outbound_event(
+    az_event_policy* policy,
+    az_event const event)
+{
+  _az_RETURN_IF_FAILED(az_event_policy_send_outbound_event(policy, event));
```

Comment:
That is because we need a function type of `az_event_policy_handler` for the handler, `az_event_policy_send_outbound_event` does not fit the signature